.changeset/hungry-insects-push.md

```diff
@@ -0,0 +1,11 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+---
+"section": tree
+---
+
+`TreeBranchEvents` exposes `rootChanged`
+
+`TreeBranchEvents` now includes the `rootChanged` event from `TreeViewEvents`.
```

Comment:
It looks like a field was removed as well, should that be called out in the changeset?

Reply:
`schemaChanged()`is also present in `TreeViewEvents` so nothing was lost.